### PR TITLE
Regenerate from `synapse-module-cookiecutter-template`

### PR DESCRIPTION
**Pull request by squahtx**
otherwise the license and description will be missing from PyPI

### CODE REVIEW

**Review by DMRobertson — APPROVED** [review]
SGTM.

How did you regenerate this---does cookiecutter let you merge a template onto an existing instance somehow?

File: setup.cfg [hunk 1]
```diff
@@ -1,8 +1,12 @@
 [metadata]
-name = matrix-python-common
+name = matrix_common
```
Comment (clokep) [inline setup.cfg:2]: Is this change wanted?

Comment (DMRobertson) [reply setup.cfg:2]: I think the intent is to make the module/package name `matrix_common` the same as the thing on PyPI?

**Comment by squahtx** [comment]
> SGTM.
> 
> How did you regenerate this---does cookiecutter let you merge a template onto an existing instance somehow?

Manually unfortunately. And not that I'm aware of.

File: setup.cfg (hunk 1 above)
Comment (squahtx) [reply setup.cfg:2]: It's neither intentional nor unintentional. I did notice the change but didn't feel too strongly about it.

It brings `setup.cfg` closer in line with the template. It does have the effect of changing the PyPI package name from `matrix-python-common` to `matrix-common`. I'm open to keeping it as `matrix-python-common`.

I think this is distinct from the `import` name / module directory name, which is already `matrix_common`. Or would be if I could figure out why the packages I build are all empty.

**Pull request merged**